src/vs/base/common/paths.ts

```diff
@@ -339,6 +342,9 @@ export function isEqualOrParent(path: string, candidate: string, ignoreCase?: bo
 		return false;
 	}
 
+	path = normalize(path);
```

Comment:
@isidorn wouldn't this have to move above the `if (candidate.length > path.length) {` check?

Reply:
@bpasero yes, I can update that